Add new issues templates (#1503)
freeCodeCamp/devdocs: .github/ISSUE_TEMPLATE/documentation_bug.md

---

> name: Documentation style bug
> about: Report documentation that has lost its style

Reviewer: ```suggestion
name: Documentation bug
about: Report a problem with a specific documentation
```

Since the problem could also be:

- documentation is missing a page
- documentation is missing a sidebar entry
- ...?
